README.md

```diff
@@ -1,2 +1,15 @@
 # sw-id-core
-Core Software Identifier library and routines for GUAC
+
+This repository contains core Software Identifier libraries and routines used
+elsewhere in the GUAC project. These are gathered here to be shared and used by
+other projects as well.
+
+## Governance
```

Comment:
We probably need a PR to guac-landing to add this repo to the [ladder](https://github.com/guacsec/guac-landing/blob/main/content/contributing.md?plain=1#L23). Is this a new topic area, or does it fall into one of the existing areas?